[2018-04-23 22:00:46,641] WARN Client session timed out, have not heard from server in 3382281ms for sessionid 0x10005498b1c0000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:46,766] INFO Client session timed out, have not heard from server in 3382281ms for sessionid 0x10005498b1c0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:46,620] WARN Client session timed out, have not heard from server in 3383750ms for sessionid 0x20005498adf0000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:46,822] INFO Client session timed out, have not heard from server in 3383750ms for sessionid 0x20005498adf0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:46,849] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Connection to node 1 could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2018-04-23 22:00:46,870] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=1856011024, epoch=INITIAL) to node 1: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:46,870] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-0=(offset=0, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1856011024, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:92)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:47,404] INFO Opening socket connection to server 127.0.0.1/127.0.0.1:12181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,411] INFO Socket connection established to 127.0.0.1/127.0.0.1:12181, initiating session (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,656] INFO Session establishment complete on server 127.0.0.1/127.0.0.1:12181, sessionid = 0x10005498b1c0000, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,698] INFO Opening socket connection to server 127.0.0.1/127.0.0.1:12181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,706] INFO Socket connection established to 127.0.0.1/127.0.0.1:12181, initiating session (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,724] INFO Session establishment complete on server 127.0.0.1/127.0.0.1:12181, sessionid = 0x20005498adf0000, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:00:47,906] WARN [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Connection to node 2 could not be established. Broker may not be available. (org.apache.kafka.clients.NetworkClient)
[2018-04-23 22:00:47,908] INFO [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=12497799, epoch=INITIAL) to node 2: java.net.SocketTimeoutException: Failed to connect within 30000 ms. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:47,908] WARN [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=1, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-2=(offset=15, logStartOffset=0, maxBytes=1048576), wing-binlog-event-1=(offset=55, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=12497799, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.net.SocketTimeoutException: Failed to connect within 30000 ms
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:92)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:48,914] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=1856011024, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:48,915] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-0=(offset=0, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1856011024, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:91)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:49,931] INFO [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=12497799, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:49,931] WARN [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=1, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-2=(offset=15, logStartOffset=0, maxBytes=1048576), wing-binlog-event-1=(offset=55, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=12497799, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:91)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:50,935] INFO [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error sending fetch request (sessionId=1856011024, epoch=INITIAL) to node 1: java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:50,935] WARN [ReplicaFetcher replicaId=2, leaderId=1, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=2, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-0=(offset=0, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=1856011024, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.0.105:9093 (id: 1 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:91)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:51,953] INFO [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error sending fetch request (sessionId=12497799, epoch=INITIAL) to node 2: java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.. (org.apache.kafka.clients.FetchSessionHandler)
[2018-04-23 22:00:51,953] WARN [ReplicaFetcher replicaId=1, leaderId=2, fetcherId=0] Error in response for fetch request (type=FetchRequest, replicaId=1, maxWait=500, minBytes=1, maxBytes=10485760, fetchData={wing-binlog-event-2=(offset=15, logStartOffset=0, maxBytes=1048576), wing-binlog-event-1=(offset=55, logStartOffset=0, maxBytes=1048576)}, isolationLevel=READ_UNCOMMITTED, toForget=, metadata=(sessionId=12497799, epoch=INITIAL)) (kafka.server.ReplicaFetcherThread)
java.io.IOException: Connection to 192.168.0.105:9094 (id: 2 rack: null) failed.
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:70)
	at kafka.server.ReplicaFetcherBlockingSend.sendRequest(ReplicaFetcherBlockingSend.scala:91)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:220)
	at kafka.server.ReplicaFetcherThread.fetch(ReplicaFetcherThread.scala:43)
	at kafka.server.AbstractFetcherThread.processFetchRequest(AbstractFetcherThread.scala:146)
	at kafka.server.AbstractFetcherThread.doWork(AbstractFetcherThread.scala:111)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2018-04-23 22:00:53,083] INFO [Partition wing-binlog-event-0 broker=1] Expanding ISR from 1 to 1,2 (kafka.cluster.Partition)
[2018-04-23 22:00:53,204] INFO [Partition wing-binlog-event-2 broker=2] Expanding ISR from 2 to 2,0 (kafka.cluster.Partition)
[2018-04-23 22:00:53,258] INFO [Partition wing-binlog-event-1 broker=2] Expanding ISR from 2 to 2,0 (kafka.cluster.Partition)
[2018-04-23 22:00:53,259] INFO [Partition wing-binlog-event-0 broker=1] Expanding ISR from 1,2 to 1,2,0 (kafka.cluster.Partition)
[2018-04-23 22:00:53,980] INFO [Partition wing-binlog-event-2 broker=2] Expanding ISR from 2,0 to 2,0,1 (kafka.cluster.Partition)
[2018-04-23 22:00:53,982] INFO [Partition wing-binlog-event-1 broker=2] Expanding ISR from 2,0 to 2,0,1 (kafka.cluster.Partition)
[2018-04-23 22:02:06,259] WARN Client session timed out, have not heard from server in 67827ms for sessionid 0x20005498adf0000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:06,751] INFO Client session timed out, have not heard from server in 67827ms for sessionid 0x20005498adf0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:06,265] WARN Client session timed out, have not heard from server in 67843ms for sessionid 0x10005498b1c0000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:06,818] INFO Client session timed out, have not heard from server in 67843ms for sessionid 0x10005498b1c0000, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:06,926] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2018-04-23 22:02:06,926] INFO [ZooKeeperClient] Connected. (kafka.zookeeper.ZooKeeperClient)
[2018-04-23 22:02:07,031] INFO Opening socket connection to server 127.0.0.1/127.0.0.1:12183. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:07,032] INFO Socket connection established to 127.0.0.1/127.0.0.1:12183, initiating session (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:07,404] INFO Session establishment complete on server 127.0.0.1/127.0.0.1:12183, sessionid = 0x10005498b1c0000, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:07,731] INFO Opening socket connection to server 127.0.0.1/127.0.0.1:12182. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:07,732] INFO Socket connection established to 127.0.0.1/127.0.0.1:12182, initiating session (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:02:07,741] INFO Session establishment complete on server 127.0.0.1/127.0.0.1:12182, sessionid = 0x20005498adf0000, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2018-04-23 22:10:01,108] INFO [GroupMetadataManager brokerId=1] Removed 0 expired offsets in 1 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:10:02,646] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:20:01,133] INFO [GroupMetadataManager brokerId=1] Removed 0 expired offsets in 2 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:20:02,677] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 1 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:30:01,146] INFO [GroupMetadataManager brokerId=1] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:30:02,688] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:40:01,176] INFO [GroupMetadataManager brokerId=1] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:40:02,721] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:50:01,192] INFO [GroupMetadataManager brokerId=1] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2018-04-23 22:50:02,737] INFO [GroupMetadataManager brokerId=2] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
